src/sonic-config-engine/portconfig.py

```diff
@@ -259,6 +260,13 @@ def __init__(self, name, bmode, properties):
         if not self._breakout_capabilities:
             raise RuntimeError("Unsupported breakout mode {}!".format(bmode))
 
+    def _parse_name_to_prefix_and_id(self):
+        match = re.match(swsscommon.FRONT_PANEL_PORT_REGEX, self._name)
+        try:
```

Comment:
>try [](http://example.com/codeflow?start=8&length=3)

Just check `match` instead of `try/except`?

Reply:
Yes, good idea, i'll do that